### PR TITLE
Shim Compose

### PR DESCRIPTION
**Pull request by ejeselsohn**

### CODE REVIEW

File: gslib/tests/test_compose.py [hunk 1]
```diff
@@ -65,33 +66,54 @@ def test_compose_too_many_fails(self):
...
-                            return_stderr=True)
-    expected_msg = ('CommandException: GCS does '
-                    'not support inter-bucket composing.\n')
-    self.assertIn(expected_msg, stderr)
+    bucket_uri_1 = self.CreateBucket()
+    bucket_uri_2 = self.CreateBucket()
+    object_uri1 = self.CreateObject(bucket_uri=bucket_uri_1, contents=b'1')
+    object_uri2 = self.CreateObject(bucket_uri=bucket_uri_2, contents=b'2')
+    components = [object_uri1, object_uri2]
+    composite = self.StorageUriCloneReplaceName(bucket_uri_1,
```
Comment (NickGoog) [inline gslib/tests/test_compose.py:86]: Maybe `composite_object_uri`

File: gslib/tests/test_compose.py [hunk 2]
```diff
@@ -65,33 +66,54 @@ def test_compose_too_many_fails(self):
...
-                            expected_status=1,
-                            return_stderr=True)
-    expected_msg = ('CommandException: GCS does '
-                    'not support inter-bucket composing.\n')
-    self.assertIn(expected_msg, stderr)
+    bucket_uri_1 = self.CreateBucket()
+    bucket_uri_2 = self.CreateBucket()
+    object_uri1 = self.CreateObject(bucket_uri=bucket_uri_1, contents=b'1')
+    object_uri2 = self.CreateObject(bucket_uri=bucket_uri_2, contents=b'2')
+    components = [object_uri1, object_uri2]
```
Comment (NickGoog) [inline gslib/tests/test_compose.py:85]: What's this for?

**Review by NickGoog — APPROVED** [review]

File: gslib/tests/test_compose.py (hunk 1 above)
Comment (ejeselsohn) [reply gslib/tests/test_compose.py]: Done.

File: gslib/tests/test_compose.py (hunk 2 above)
Comment (ejeselsohn) [reply gslib/tests/test_compose.py]: Deleted.

**Pull request merged**